src/components/victory-candlestick/candle.js

```diff
@@ -0,0 +1,68 @@
+import React, { PropTypes } from "react";
+import VictoryCandlestick from "./victory-candlestick";
+
+
+export default class Candle extends React.Component {
+  static propTypes = {
+    index: React.PropTypes.number,
+    x: PropTypes.number,
+    y1: PropTypes.number,
+    y2: PropTypes.number,
+    candleColor: PropTypes.string,
+    y: PropTypes.number,
+    candleHeight: PropTypes.number,
+    scale: PropTypes.object,
+    style: PropTypes.object,
+    datum: PropTypes.object,
+    width: PropTypes.number,
+    padding: PropTypes.number,
+    data: PropTypes.array
+  }
+
+  renderWick() {
+    const width = VictoryCandlestick.defaultProps.width;
+    const padding = VictoryCandlestick.defaultProps.padding;
+    const dataLength = this.props.data.length;
+    const x = this.props.x + 0.25 * (width - 2 * padding) / dataLength;
+
+    return (
+        <line
+          x1={x}
+          x2={x}
+          y1={this.props.y1}
+          y2={this.props.y2}
+          stroke={this.props.candleColor}
+          strokeWidth={1}
+        />
+      );
+  }
+
+  renderCandle() {
+    const width = VictoryCandlestick.defaultProps.width;
```

Comment:
This is an antipattern. Pass width and padding down as props to candle rather than referencing VictoryCandlestick here. Also, you should never need to reference defaultProps directly. These will just always show up in props, and that gives the user a change to override the defaults with passed props.